shared/reducers/tracker.js

```diff
@@ -404,6 +404,14 @@ export default function(
         }))
       }
     }
+    case TrackerGen.updateStellarAddress: {
+      const {federationAddress, accountID, sigID} = action.payload
+      console.log('::::::: Got TrackerGen.updateStellarAddress: ', federationAddress, accountID, sigID)
```

Comment:
🔥

Reply:
ah good catch, sorry, i keep leaving this stuff around :disappointed_relieved: